styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java

```diff
@@ -129,20 +139,35 @@
   }
 
   @Override
-  public String createExecution(final RunState runState, final String execName, final FlyteExecConf flyteExecConf,
-                                final Map<String, String> annotations)
+  public String createExecution(final RunState runState, final String execName, final FlyteExecConf flyteExecConf)
       throws CreateExecutionException {
     requireNonNull(runState, "runState");
     requireNonNull(execName, "name");
     requireNonNull(flyteExecConf, "flyteExecConf");
-    requireNonNull(annotations, "annotations");
     final var launchPlanIdentifier = flyteExecConf.referenceId();
     final var execMode = runState.data().trigger()
         .map(this::toFlyteExecutionMode)
         .filter(mode -> mode != ExecutionMode.UNRECOGNIZED)
         .orElseThrow(() -> new CreateExecutionException("Missing trigger or unknown in StateData: " + runState.data()));
 
-    var extraDefaultInputs = getExtraDefaultInputs(runState.workflowInstance(), runState.data());
+    var styxVariables = getStyxVariables(runState.workflowInstance(), runState.data());
+    var triggeredParams = runState.data().triggerParameters()
+        .map(FlyteAdminClientRunner::getFilteredTriggerParams)
+        .orElseGet(Map::of);
+
+    var labels = Map.of(STYX_EXECUTION_ID, normalize(styxVariables.get(STYX_EXECUTION_ID)));
```

Comment:
I guess `normalize` function won't do anything so we will get the same string back, right?

Reply:
Yes I can use just the string and add a comment that normalize in this case is not necessary because the way we generate styx_execution ids. I can add a test to make sure that we don't break that invariant